bors.toml

```diff
@@ -2,7 +2,7 @@ status = ["Build", "Lint style", "Post-Build Step"]
 use_squash_merge = true
 timeout_sec = 28800
 block_labels = ["WIP", "blocked-by-other-PR", "merge-conflict", "awaiting-CI", "bench-after-CI"]
-delete_merged_branches = true
+delete_merged_branches = false
 update_base_for_deletes = true
```

Comment:
Isn't this setting supposed to prevent the PR closures?

Reply:
Good point... I'll dig into the bors code then.